src/frame.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+function Frame(game) {
+  this.score = []
+}
+
+Frame.prototype.bowlFrame = function(game){
+  var bowl = Math.floor((Math.random() * 11));
```

Comment:
there was no requirement to randomly generate the values passed to the score calculator.  If there was, then this would still be a questionable design; ideally you should separate the generation of the random number from storing it in the frame.